hw/ip/hmac/data/hmac.prj.hjson

```diff
@@ -10,3 +10,11 @@
     verification_stage: "V3",
     notes:              "D3 commit id: 3471a47c96ba87c77de7b14c39f4faf71a839283",
 }
+{
+    name:               "hmac",
+    version:            "0.6",
+    life_stage:         "L1",
+    design_stage:       "D2",
+    verification_stage: "V1",
+    notes:              "",
+}
```

Comment:
@sjgitty Is this the correct way to do that?

Reply:
No, please see uart.prj.hjson. It requires a revisions list.

Reply:
ah got it, thanks! amended.